docs/fleet/fragments.md

```diff
@@ -9,6 +9,10 @@ aliases:
   - /fleet/configure-a-fleet/
```

Comment:
Please also consider what here now needs to be removed. I don't want a followup ticket for that

Reply:
done